app/controllers/users_controller.rb

```diff
@@ -11,16 +18,60 @@ def create
     @user = User.new user_params
     if @user.save
       log_in @user
-      flash[:success] = t ".welcome"
+      flash[:success] = t "users.create.welcome"
       redirect_to @user
     else
       render :new
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update_attributes user_params
+      flash[:success] = t "users.update.mess_success"
+      redirect_to @user
+    else
+      flash.now[:danger] = t "users.update.mess_fail"
+      render :edit
+    end
+  end
+
+  def destroy
+    @user.destroy
+      if @user.destroyed?
+        flash[:success] = t "users.destroy.mess_success"
+      else
+        flash[:danger] = t "users.destroy.mess_fail"
+      end
+      redirect_to users_url
+  end
+
+  def logged_in_user
+    return if logged_in?
```

Comment:
dùng unless thay cho return if nhé

Reply:
Ban đầu em  dùng unless mà bé rubocop bắt lỗi " Use a guard clause instead of wrapping the code inside a conditional expression" nên em lại phải dùng return if ấy chị.hjx